### ISSUE TITLE
feat: Custom aria-label for menu button

### ISSUE THREAD

**Issue by gurpreetonebcg** [ionic-team/ionic-framework#23604]
# Feature Request

**Ionic version:**

```
@ionic/angular 5.6.0
```

**Describe the Feature Request**

aria-label for the menu button should be custom

**Describe Preferred Solution**

aria-label for the menu button should be custom, so that developer can change the value of the aria-label. Request is raised as per the WCAG guide line 3.1.2 Language of Parts, so that developers can use ML in aria-label to satisfy the above guide line.

**Additional Context**

[Screen shorts of issues](https://drive.google.com/file/d/1cOTmk8LBzmUsxYTM8FzH8Agao9RHpGSO/view?usp=sharing)

**Comment by liamdebeasi** [comment]
Thanks for the issue. This has been resolved via https://github.com/ionic-team/ionic-framework/pull/23608, and a fix will be available in an upcoming release of Ionic Framework.
